### DISCUSSION TITLE
"Headless" Tsynth ?

### DISCUSSION THREAD

**Discussion by fitzgreyve** [ElectroTechnique/TSynth-Teensy4.1#138] [T4.1 code development]
Hi there,
I'd like to try and modify the code to make a "headless" version  - i.e. no controls (pots, switches, encoder), no display, just the teensy with MIDI in and audio out.

before I start working through the code, any ideas on whcih might be the "best" (low level?) code modules to look at to disable/ignore the hardware ?

regards,
Mark

**Comment by winder** [comment]
I don' think you need to modify the software for it to work the way you want.

If you really want to completely remove the physical i/o option, [here is a good place to start](https://github.com/ElectroTechnique/TSynth-Teensy4.1/blob/103f6b2f85b6198e714062ced98916384886d92b/TSynth/TSynth.ino#L1837).
